sdk/iot/hub/samples/src/sample_sas_utility.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "az_result.h"
+#include "az_span.h"
+
+/*
+ * This serves as an example for fundamental functionality needed to use SAS key authentication.
+ * This implementation uses OpenSSL.
```

Comment:
this is header file.... Can we make this comment in c file?

Reply:
moved